src/Illuminate/Foundation/Testing/Concerns/InteractsWithConsole.php

```diff
@@ -48,6 +55,18 @@ public function artisan($command, $parameters = [])
                 $this->fail('Question "'.Arr::first($this->expectedQuestions)[0].'" was not asked.');
             }
 
+            if (count($this->expectedChoices) > 0) {
+                foreach ($this->expectedChoices as $question => $answers) {
+                    $assertion = $answers['strict'] ? 'assertEquals' : 'assertEqualsCanonicalizing';
+
+                    $this->$assertion(
+                        $answers['expected'],
+                        $answers['actual'],
+                        'Question "'.$question.'" has different options.'
```

Comment:
Wouldn't it be convenient to show the actual different options?

Reply:
since we compare arrays phpunit will display the diff like so
```bash
Question "Please pick a model" has different options.
Failed asserting that two arrays are equal.
--- Expected
+++ Actual
@@ @@
 Array (
-    0 => 'A'
-    1 => 'B'
+    0 => '<href=file:///home/adrian/code/laravel-factories-reloaded/example/app/Models/Group.php>ExampleApp\Models\Group</>'
+    1 => '<href=file:///home/adrian/code/laravel-factories-reloaded/example/app/Models/Ingredient.php>ExampleApp\Models\Ingredient</>'
+    2 => '<href=file:///home/adrian/cod...ipe</>'
+    3 => 'All'
 )
```